src/database/migrations/20240513091857-create-reviews-table.js

```diff
@@ -27,7 +27,8 @@ module.exports = {
 					'APPROVED',
 					'REJECTED',
 					'PUBLISHED',
-					'REJECTED_AND_REPORTED'
+					'REJECTED_AND_REPORTED',
+					'CHANGES_UPDATED'
```

Comment:
this migration is already executed right? @priyanka-TL

Reply:
Updated in old migration we can add this directly